circuitpython_build_tools/build.py

```diff
@@ -100,64 +100,31 @@ def _munge_to_temp(original_path, temp_file, library_version):
                 temp_file.write(line.encode("utf-8") + b"\r\n")
     temp_file.flush()
 
-def library(library_path, output_directory, mpy_cross=None, example_bundle=False, pkg_folder_prefix=None):
+def library(library_path, output_directory, mpy_cross=None, example_bundle=False):
     py_files = []
     package_files = []
     example_files = []
     total_size = 512
     for filename in os.listdir(library_path):
         full_path = os.path.join(library_path, filename)
-        if os.path.isdir(full_path):
```

Comment:
Ok, I suspect that this isn't doing what it really was intending, which is to walk the directory structure looking for python .py files

```
excludes=[ "docs", "examples" ]
walked_files = []
root_dir = os.getcwd()
for root, dirs, files in os.walk( root_dir ):
    dirs[:] = [d for d in dirs if d not in excludes]
    for file in files:
        rel_dir = os.path.relpath(root, root_dir)
        walked_files.append( os.path.join(rel_dir, file) )
files = filter(lambda x: x.endswith(".py") or x.startswith("font5x8.bin"), walked_files)
print files
```
seems like it's a closer solution to what was being attempted here